src/codeflare_sdk/utils/kube_api_helpers.py

```diff
@@ -42,3 +43,7 @@ def _kube_api_error_handling(e: Exception):  # pragma: no cover
         elif e.reason == "Conflict":
             raise FileExistsError(exists_msg)
     raise e
+
+
+def _get_api_host(api_client: client.ApiClient):  # pragma: no cover
```

Comment:
When creating the appwrapper while authenticated with a K8s config file I experienced an error `'NoneType' object has no attribute 'configuration'`

When I ran through these changes while authenticated with token and server and it ran as expected. Just need to fix the _get_api_host function